docs/templates.md

```diff
@@ -99,6 +99,31 @@ $pagination->setCustomParameters([
 ]);
 ```
 
+If you want to use the template with the same parameters everywhere, create your own
+template and let extend it from the provided template.
+
+Here's an example how to achieve that
+
+```yaml
+knp_paginator:
+    template:
+        pagination: 'pagination.html.twig'
+```
+
+And in your template
+
+```twig
+{% set align = 'left' %}
+{% set size = 'large' %}
+{% set style = 'bottom' %}
+
+{% extends '@KnpPaginator/Pagination/twitter_bootstrap_v4_pagination.html.twig' %}
+```
+
+Alternatively you can override the origin template in your `templates/bundles/`
```

Comment:
Missing comma after "Alternatively". 
"origin" should be "original".

Reply:
Thanks! I have fixed in the new commit,